pkgs/by-name/ca/cambalache/package.nix

```diff
@@ -18,21 +18,33 @@
   webkitgtk_4_1,
   webkitgtk_6_0,
   nix-update-script,
+  libepoxy,
+  wayland-scanner,
+  wlroots_0_18,
+  libxkbcommon,
 }:
 
+let
+  casilda = fetchFromGitLab {
```

Comment:
Looks like this can be used like a regular package, would you mind packaging it?

https://gitlab.gnome.org/jpu/casilda

We prefer non-vendored packages where possible as, among other things, it allows performing maintenance work just once rather than in each dependent, and it requires redoing less work when it fails in the later stages of the build.